cloudigrade/account/management/commands/add_account.py

```diff
@@ -0,0 +1,8 @@
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'This adds aws accounts'
+
+    def handle(self, *args, **options):
+        self.stdout.write(self.style.SUCCESS('AWS Account Added... probably'))
```

Comment:
Since this command doesn't actually do anything yet, let's change the message to a more _obvious_ dummy placeholder. Maybe something like "AWD Account command placeholder message". I'm also a big fan of `# FIXME` comments to make it super obvious to readers that things aren't working as expected.